tests/rpm-ostree/main.yml

```diff
@@ -0,0 +1,571 @@
+---
+# vim: set ft=ansible:
+#
+#   This is a general test suite for rpm-ostree that exercises several of the
+# rpm-ostree subcommands.  These tests assume the atomic host is provisioned
+# with the latest version.  These tests are meant to be run in succession.
+#
+- name: rpm-ostree - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    # This playbook requires Ansible 2.2 and an Atomic Host
+    - role: ansible_version_check
+      avc_major: "2"
+      avc_minor: "2"
+      tags:
+        - ansible_version_check
+
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+#####################################################################################
+#
+# rpm-ostree deploy version
+#   - validates deploying by version string
+#   - validates cleanup of rollback deployment
+#
+#####################################################################################
+
+- name: rpm-ostree - deploy by version
+  hosts: all
+  become: yes
+
+  tags:
+    - deploy_version
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - name: Get current refspec
+      shell: cat $(ostree admin --print-current-dir).origin | grep refspec | cut -d= -f2
```

Comment:
Can we get this from the JSON output of `rpm-ostree status`?  It looks like it may be the `origin` value:

```
# rpm-ostree status --json | python -m json.tool
{
    "deployments": [
        {
            "booted": true,
            "checksum": "0ccf9138962e5c2c3794969a228e751d13bb780f5b0a1f15f4a9649df06ba80a",
            "gpg-enabled": false,
            "id": "rhel-atomic-host-0ccf9138962e5c2c3794969a228e751d13bb780f5b0a1f15f4a9649df06ba80a.0",
            "origin": "rhel-atomic-host:rhel-atomic-host/7/x86_64/standard",
            "osname": "rhel-atomic-host",
            "packages": [],
            "regenerate-initramfs": false,
            "requested-local-packages": [],
            "requested-packages": [],
            "serial": 0,
            "timestamp": 1495490444,
            "unlocked": "none",
            "version": "7.3.5"
        }
    ],
    "transaction": null
}
```

@jlebon Can you confirm this?